### PR TITLE
Feat/minimize drawer

### PR DESCRIPTION
**Pull request by trungnq2k**

### CODE REVIEW

**Comment by trungnq2k** [comment]
# Description

- Add sliding animation for minimizing drawer
- Replace chakra-ui drawer useDisclosure() hook with CSS transformation, preventing drawer component from being unmounted
- Reorganize space speakers and popular spaces layout when minimizing drawer

image
image

File: pho/src/components/DrawerComponent.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+.drawer-close {
+    align-items: flex-end !important;
+    transform:  translateX(-350px) !important;
+}
+
+.drawer-open {
+    /* align-items: inl !important; */
+    transform:  translateX(0px) !important
+}
```
Comment (richardtlp) [inline pho/src/components/DrawerComponent.css]: Just a small thing: reformat the css to 2 spaces.

**Review by richardtlp — APPROVED** [review]

**Comment by chaule-mck** [comment]
image

@trungnq2k when minimizing signup/signin components, either hide the component completely or display the minimized version

File: pho/src/components/PopularSpace/SpaceCell/SpaceCell.tsx [hunk 2]
```diff
@@ -17,14 +17,23 @@ import noOnlUsersIcon from '@/asset/noOnlUsersIcon.png';
...
+
+const SpaceCell: React.FC<SpaceCellProps> = ({ isDrawerOpen, space }) => {
   const { isOpen, onOpen, onClose } = useDisclosure();
   const URL = process.env.REACT_APP_BACKEND_URL ?? '';
   const [spaceInfo, setSpaceInfo] = useState<GetSpaceDetailResponse>();
 
   useEffect(() => {
     const fetchSpaceInfo = async () => {
       const spaceInfo = (await axios.get(`${URL}/api/spaces/${space.id}`)).data;
+      console.log(spaceInfo);
```
Comment (thienanhvu) [inline pho/src/components/PopularSpace/SpaceCell/SpaceCell.tsx:35]: I think we should remove all unneccessary log to advoid confusing while debug

**Review by thienanhvu — APPROVED** [review]

File: pho/src/components/DrawerComponent.css (hunk 1 above)
Comment (trungnq2k) [reply pho/src/components/DrawerComponent.css]: Resolved!

File: pho/src/components/PopularSpace/SpaceCell/SpaceCell.tsx (hunk 2 above)
Comment (trungnq2k) [reply pho/src/components/PopularSpace/SpaceCell/SpaceCell.tsx:35]: Resolved!

**Pull request merged**